packages/gatsby-theme-api-docs/gatsby-config.js

```diff
@@ -1,17 +1,6 @@
 const path = require('path');
 
-const getTagListOption = (themeOptions) => {
```

Comment:
Linter complained about unused code and was right. This was left over from the mdx introspection plugin